bench/benchmarks_tf.py

```diff
@@ -0,0 +1,162 @@
+import tensorflow as tf
```

Comment:
I think you will need to add the Meta copyright header so that we don't get an automated warning later on.